srml/aura/Cargo.toml

```diff
@@ -16,11 +16,17 @@ timestamp = { package = "srml-timestamp", path = "../timestamp", default-feature
 staking = { package = "srml-staking", path = "../staking", default-features = false }
 session = { package = "srml-session", path = "../session", default-features = false }
 
+[dependencies.aura-primitives]
```

Comment:
Why not use the regular syntax we use everywhere? Is `version` actually needed?

Reply:
`version` is for if we want to publish on `crates.io`.  The different syntax is to avoid CI complaining about line length.

Reply:
None of the other internal crates have `version` here, so we would need to rework this module anyway before publish. I don't think we should introduce `version` just yet.

I think you can break the regular syntax into multiple lines without issues:
```toml
aura-primitives = {
  package = "substrate-consensus-aura-primitives",
  path = "../../...",
  default-features = false,
}
```

Reply:
> None of the other internal crates have `version` here, so we would need to rework this module anyway before publish. I don't think we should introduce `version` just yet.
> 

Will fix, thanks!

> I think you can break the regular syntax into multiple lines without issues:
> 
> ```toml
> aura-primitives = {
>   package = "substrate-consensus-aura-primitives",
>   path = "../../...",
>   default-features = false,
> }
> ```

That isn’t valid TOML, although it probably should be.